trainingRN/src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createStackNavigator } from 'react-navigation';
+import Todo from './screens/todo';
+import BookList from './screens/bookList';
```

Comment:
Remember that folders that belong to React `Components` should be written with capital letters:
```
./screens/BookList
./screens/Todo
```